package.json

```diff
@@ -23,6 +23,7 @@
     "lib",
     "cli.js"
   ],
+  "typings": "./d.ts/index.d.ts",
```

Comment:
Thanks @hzlmn 👍  

What if I am not using TypeScript and just want my editor to pickup the types for intelligent code suggestions?